### ISSUE TITLE
[False Positive] Killaura

### ISSUE THREAD

**Issue by MithrandirCraft** [CraftG/Horizon#60]
You can easily reproduce killaura false positives by just spawning some villagers in a clean spigot server with WorldEdit, Horizon, Essentials & ProtocolLib and hitting them. All plugins and server are freshly installed.

You don't even need to spawn test mobs for this purpose, natural mobs in wilderness can trigger this quite easy too. I got two killaura false positives (Shown in video) in 10 minutes.

INFOMATION:
Server: (PaperSpigot/Spigot/Torch/...)
Server version: 1.13.2

bypass/false positive/bug/error/suggestion
ISSUE TYPE: False Positive

If reporting a false positive/bypass. (If possible)
VIDEO: https://youtu.be/_0o9hLkELek

**Edit:** Disclaimer, In the video, I am using Skillclient, but KillAura is inactive. I'm just hitting villagers.

**Comment by CraftG** [comment]
Hello. I'm not sure if it's fixed in the latest version because I can't reproduce it. You can close this issue if it's fixed. If not, please use /hori verbose to enable verbose and upload the output.

**Comment by BestMark** [comment]
Try to fight without SkillClient. I think, there is some modifications in it, even if KillAura is inactive.

**Comment by MithrandirCraft** [comment]
Understood, will report back trying both.

**Comment by MithrandirCraft** [comment]
Ok. So I tried Horizon without using any hacked clients, clean 1.4.5 install (latest release).
I hit villagers for about 7 minutes and couldn't reproduce any false positives.
I then opened skillclient, activated killaura and started hitting villagers, at first I hit villagers with killaura for about a minute (looks pretty slow, and I've already seen some reports about it), after that I god inmediately detected for killaura while eating (I find this fair enough, it's a safe way to confirm killaura usage).

As long as it detects killaura eventually, without false positives, it's good enough for me. I prefer for it to be a slow but reliable detection than having tons of false positives.

**Comment by MithrandirCraft** [comment]
Ok, so testing this on my server, where only 3 people are playing, I got quite a few killaura false positives. The best way to reproduce them are apparently just hitting mobs in a farm spawner.
It takes some people more effort than others, but they all eventually get a few killaura false positives.
Would like to record this, but it's pretty much the same as in the shown video... it's not consistent, so there's no way I can tell you to do X thing to reproduce. 

Best I can say is, try it yourselves on a crowded server, because the killaura checks definetly generate false positives.

For now I hope the feedback I've provided is usefull, I must also add, I've tested many hack blockers (paid and unpaid), and I can say Horizon is just as unsuccessfull at efficiently blocking killaura as the rest. I only wish the best for the future of this plugin, but I'm afraid unless things change I won't be using it further.
